.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
-  - "6.10.2"
+  - "6.10.3"
+  - "7"
+  - "8"
```

Comment:
If we do this, we have to make sure npm deployment only happen once on a specific Node version. Otherwise, let's not add Node 7 and 8 builds to this PR yet.

Reply:
Great call! This is not how I expected this feature to work. I've updated it per [this](https://docs.travis-ci.com/user/deployment/#Examples-of-Conditional-Deployment) example.